### ISSUE TITLE
Lien vers une cagnotte

### ISSUE THREAD

**Issue by patou** [patou/liste-envies#64]
Ajouter un lien vers sur cagnotte sur une liste d'envie.
Cela ajoute une icone avec un lien vers la cagnotte.

**Comment by patou** [comment]
Fait

**Comment by manudss** [comment]
Oui, on peut ajouter un lien, mais on peut faire quelque chose de plus intégré. Par exemple, mettre le liens vers la cagnotte, directement dans la config de la liste. et même faire la possibilité de payer directement lorsque l'on fait offrir. Paypal, permet de créer un lien pour payer directement : [http://paypal.me/EDESAINTSTEBAN/650](http://paypal.me/EDESAINTSTEBAN/650 ) ou tu peux spécifier le montant.

**Comment by patou** [comment]
paypal.me, c'est pour faire un virement vers une personne, il faudrait plutôt arriver à créer une vrai cagnote paypal.
